templates/rhpam75-authoring-ha.yaml

```diff
@@ -75,7 +75,7 @@ parameters:
   generate: expression
   required: false
 - displayName: KIE Server Bypass Auth User
-  description: Allows the KIE server to bypass the authenticated user for task related operations e.g. queries. (Sets the org.kie.server.bypass.auth.user system property)
+  description: Allows the KIE server to bypass the authenticated user for task related operations, for example, queries. (Sets the org.kie.server.bypass.auth.user system property)
```

Comment:
Not the type of thing you're after, but since "task related operations" appears so many times, just noting that it should really be "task-related operations". But not substantive.